docs/src/modules/zend.service-manager.intro.rst

```diff
@@ -17,144 +22,179 @@ Following is the ``Zend\ServiceManager\ServiceLocatorInterface`` API:
        public function get($name);
        public function has($name);
    }
-	
+
 - ``has($name)``, tests whether the ``ServiceManager`` has a named service;
 
 - ``get($name)``, retrieves a service by the given name.
 
+.. _zend.service-manager.api:
+
+Additional API
+--------------
+
 A ``Zend\ServiceManager\ServiceManager`` is an implementation of the ``ServiceLocatorInterface``.
 In addition to the above described methods, the ``ServiceManager`` provides additional API:
 
-- **Service registration**. ``ServiceManager::setService`` allows you to register an object as a service:
+.. _zend.service-manager.api.service-registration:
 
-  .. code-block:: php
-     :linenos:
+Service registration
+^^^^^^^^^^^^^^^^^^^^
 
-     $serviceManager->setService('my-foo', new stdClass());
-     $serviceManager->setService('my-settings', array('password' => 'super-secret'));
+``ServiceManager::setService`` allows you to register an object as a service:
 
-     var_dump($serviceManager->get('my-foo')); // an instance of stdClass
-     var_dump($serviceManager->get('my-settings')); // array('password' => 'super-secret')
+.. code-block:: php
+   :linenos:
 
-- **Lazy-loaded service objects**. ``ServiceManager::setInvokableClass`` allows you to tell the
-  ``ServiceManager`` what class to instantiate when a particular service is requested:
+    $serviceManager->setService('my-foo', new stdClass());
+    $serviceManager->setService('my-settings', array('password' => 'super-secret'));
 
-  .. code-block:: php
-     :linenos:
+    var_dump($serviceManager->get('my-foo')); // an instance of stdClass
+    var_dump($serviceManager->get('my-settings')); // array('password' => 'super-secret')
 
-     $serviceManager->setInvokableClass('foo-service-name', 'Fully\Qualified\Classname');
+.. _zend.service-manager.api.lazy-loaded-objects:
 
-     var_dump($serviceManager->get('foo-service-name')); // an instance of Fully\Qualified\Classname
+Lazy-loaded service objects
```

Comment:
The problem with using this term to discuss invokables is that we *also* have a concept of "lazy services". These are services that are wrapped in a proxy so that they only fully initialize on first invocation. I think naming this section "Lazy-loaded service objects" will make that unclear, and cause confusion.